transmogrifier/sources/xml/datacite.py

```diff
@@ -54,7 +55,8 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
                 if self.valid_content_types([content_type]):
                     fields["content_type"] = [content_type]
                 else:
-                    return None
+                    message = f'Record skipped based on content type: "{content_type}"'
+                    raise SkippedRecordEvent(message, source_record_id)
```

Comment:
👍 

I might suggest as we revist raising these in the future that we have a convention of positional `message` and then named arguments (e.g. `source_record_id=source_record_id`), but not blocking now as this definitely works too!

Reply:
I like this idea, too!

Reply:
👍